plugins/registry.js

```diff
@@ -584,4 +584,60 @@ export default class PluginRegistry {
 
         return {id, showRHSPlugin: showRHSPlugin(id), hideRHSPlugin: hideRHSPlugin(id), toggleRHSPlugin: toggleRHSPlugin(id)};
     }
+
+    // Register a Needs Team component by providing a route past /:team/ to be displayed at.
+    // Accepts the following:
+    // - route - The route to be displayed at.
+    // - component - A react component to display.
+    // Returns:
+    // - id: a unique identifier
+    registerNeedsTeamComponent(route, component) {
```

Comment:
For consistency with the other one: 
```suggestion
    registerNeedsTeamRoute(route, component) {
```